### PR TITLE
docs(jsdoc): JWT Auth Middleware

### PR DESCRIPTION
**Pull request by goisaki**
This PR is to add JSDoc for JWT Auth Middleware.
Note that the target of the PR is not `main`.

Related:
- #1338
- #2680

- [x] `bun denoify` to generate files for Deno
- [x] `bun run format:fix && bun run lint:fix` to format the code

### CODE REVIEW

**Comment by yusukebe** [comment]
@goisaki 

The title is `JSX Renderer`, but the code is JWT Middleware. Can you check ti?

**Comment by goisaki** [comment]
Aah! My mistake completely. I will correct it.

File: deno_dist/middleware/jwt/index.ts [hunk 1]
```diff
@@ -13,6 +13,33 @@ declare module '../../context.ts' {
   }
 }
 
+/**
+ * JWT Auth middleware for Hono.
+ *
+ * @see {@link https://hono.dev/middleware/builtin/jwt}
+ *
+ * @param {object} options - The options for the JWT middleware.
+ * @param {string} options.secret - A value of your secret key.
```
Comment (yusukebe) [inline deno_dist/middleware/jwt/index.ts:22]: Is the following better?

```ts
@param {string} [options.secret] - A value of your secret key.
```

Comment (yusukebe) [reply deno_dist/middleware/jwt/index.ts:22]: @goisaki 

I added by myself: https://github.com/honojs/hono/pull/2709/commits/9d1cdf73a4dd38b5bb6245edf42aaef925bb33cd

Comment (goisaki) [reply deno_dist/middleware/jwt/index.ts:22]: Thanks!

**Pull request merged**